exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -185,11 +185,10 @@
       </div>
   </div>
    <script charset="utf-8">
-      var refreshTime = 2000;
+      var refreshTime = 10000;
       var refresh = getRefreshTime();
```

Comment:
This is confusing isn't it? The function does not return a value and yet the return value (<undefined>) is stored in a variable that is not used. 
Luckily it's not used anywhere.